### PR TITLE
remove references to kubevirt-config ConfigMap

### PR DESCRIPTION
**Pull request by ashleyschuett**
This update shows the new way to configure kubevirt based on the PR https://github.com/kubevirt/kubevirt/pull/3455

### CODE REVIEW

File: installation/authorization.md [hunk 1]
```diff
@@ -42,14 +42,13 @@ all KubeVirt resources, including the ability to delete collections of
...
-**kubevirt-config** in the namespace the KubeVirt components are
-running.
+runtime config. This config exists within the Kubevirt Custom Resource under
+the `configuration` key in the namespace the KubeVirt operator is running.
 
 > *NOTE* Users are only guaranteed the ability to modify the kubevirt
 > runtime configuration if a ClusterRoleBinding is used. A RoleBinding
-> will work to provide kubevirt-config access only if the RoleBinding
-> targets the same namespace the kubevirt-config exists in.
+> will work to provide kubevirt CR access only if the RoleBinding
```
Comment (ashleyschuett) [inline installation/authorization.md:50]: Im not entirely sure what this means, so im not sure I have edited correctly. Does it make sense?

File: creation/disks-and-volumes.md [hunk 2]
```diff
@@ -431,7 +431,7 @@ those PVCs.
 
 In order to take advantage of the DataVolume volume source on a VM or
 VMI, the **DataVolumes** feature gate must be enabled in the
-**kubevirt-config** config map before KubeVirt is installed. CDI must
+**kubevirt** custom resource before KubeVirt is installed. CDI must
```
Comment (ashleyschuett) [inline creation/disks-and-volumes.md:434]: I'm unsure if this is still true

File: creation/dedicated-cpu.md [hunk 3]
```diff
@@ -146,34 +146,42 @@ running.
...
+`kubectl edit kubevirt kubevirt -n kubevirt`
 
+```
     ...
-    data:
-      feature-gates: "DataVolumes,CPUManager"
+    spec:
+      configuration:
+        developerConfiguration:
+          featureGates: "DataVolumes,CPUManager"
```
Comment (rmohr) [inline creation/dedicated-cpu.md:176]: That should probably be an array too.

File: creation/disks-and-volumes.md (hunk 2 above)
Comment (rmohr) [reply creation/disks-and-volumes.md:434]: Let's check the virt-controller and virt-api logic explicitly, but I think it is no longer true.

File: installation/installation.md [hunk 4]
```diff
@@ -42,30 +42,39 @@ virtualization workloads:
...
+[software emulation fallback](https://github.com/kubevirt/kubevirt/blob/master/docs/software-emulation.md)
+can be enabled using:
+
+    $ curl -sL https://github.com/kubevirt/kubevirt/releases/download/${VERSION}/kubevirt-cr.yaml -o kubevirt.yaml
+
+Add the following to the `kubevirt.yaml` file
+    ```
+    spec:
+    ...
+    configuration:
```
Comment (rmohr) [inline installation/installation.md:71]: Is there another level of indent necessary?

File: installation/installation.md [hunk 5]
```diff
@@ -42,30 +42,39 @@ virtualization workloads:
...
+
+    $ curl -sL https://github.com/kubevirt/kubevirt/releases/download/${VERSION}/kubevirt-cr.yaml -o kubevirt.yaml
+
+Add the following to the `kubevirt.yaml` file
+    ```
+    spec:
+      ...
+      configuration:
+        developerConfiguration:
+          useEmulation: "true"
```
Comment (rmohr) [inline installation/installation.md:73]: I think this is now a boolean.

**Comment by rmohr** [comment]
@ashleyschuett I am a little bit reluctant advertising the new way of configuring kubevirt until we properly test it. Right now we still run all tests with the config-map approach. Do you think that we can move some tests over to use the new  way?

File: installation/node-eviction.md [hunk 6]
```diff
@@ -127,20 +127,21 @@ solution.
...
-    data:
-      feature-gates: "LiveMigration"
-      migrations: |-
-        nodeDrainTaintKey: mytaint/drain
+    spec:
+      configuration:
+        developerConfiguration:
+          featureGates:
+            - "LiveMigration"
+          migrationConfiguration:
```
Comment (kbidarkar) [inline installation/node-eviction.md:143]: This indentation is not required. I guess `migrationConfiguration` should have same indent as `developerConfiguration`

File: installation/authorization.md (hunk 1 above)
Comment (rmohr) [reply installation/authorization.md:50]: I am not sure whit this section is there at all. Only admins should modify it.

**Comment by rmohr** [comment]
@ashleyschuett would you mind doing a final rebase (without my accidental commit via the github ui)? I think we are finally good to merge this.

**Comment by fabiand** [comment]
@ashleyschuett hey - please rebase and give me a ping,t hen we can get this merged.

**Review by rmohr — APPROVED** [review]
/lgtm
/approve

**Pull request merged**